projects/incubator/src/eventsystem/cms/cms-event.builder.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Action, ActionsSubject } from '@ngrx/store';
+import { CmsActions } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CmsEventBuilder {
+  constructor(protected actionsSubject: ActionsSubject) {}
```

Comment:
Maybe we could create a generic / abstract events service that can have `ActionsSubject` injected. This abstraction would be useful in case we move away from ngrx in the future. 

We can maybe consider moving `create` and `actions` methods to the generic / abstract service as well.